hibernate-core/src/main/java/org/hibernate/engine/jdbc/internal/JdbcCoordinatorImpl.java

```diff
@@ -357,71 +355,6 @@ public void disableReleases() {
 		releasesEnabled = false;
 	}
 
-	protected void close(Statement statement) {
-		LOG.tracev( "Closing prepared statement [{0}]", statement );
-
-		// Important for Statement caching -- some DBs (especially Sybase) log warnings on every Statement under
-		// certain situations.
-		sqlExceptionHelper().logAndClearWarnings( statement );
-
-		if ( statement instanceof InvalidatableWrapper ) {
-			@SuppressWarnings("unchecked")
-			final InvalidatableWrapper<Statement> wrapper = (InvalidatableWrapper<Statement>) statement;
-			close( wrapper.getWrappedObject() );
-			wrapper.invalidate();
-		}
-		else {
-			try {
-				// if we are unable to "clean" the prepared statement,
-				// we do not close it
-				try {
-					if ( statement.getMaxRows() != 0 ) {
-						statement.setMaxRows( 0 );
-					}
-					if ( statement.getQueryTimeout() != 0 ) {
-						statement.setQueryTimeout( 0 );
-					}
-				}
-				catch( SQLException sqle ) {
-					// there was a problem "cleaning" the prepared statement
-					if ( LOG.isDebugEnabled() ) {
-						LOG.debugf( "Exception clearing maxRows/queryTimeout [%s]", sqle.getMessage() );
-					}
-					// EARLY EXIT!!!
-					return;
-				}
-				statement.close();
-				if ( lastQuery == statement ) {
-					lastQuery = null;
-				}
-			}
-			catch ( Exception e ) {
-				LOG.debugf( "Unable to release JDBC statement [%s]", e.getMessage() );
-			}
-		}
-	}
```

Comment:
the above method was copied to https://github.com/hibernate/hibernate-orm/blob/de3408662cb65201c3aa4e64f032ecd44b61e804/hibernate-core/src/main/java/org/hibernate/resource/jdbc/internal/ResourceRegistryStandardImpl.java#L183